concordium-sdk/src/main/java/com/concordium/sdk/responses/peerStats/PeerStat.java

```diff
@@ -0,0 +1,44 @@
+package com.concordium.sdk.responses.peerStats;
+
+import concordium.ConcordiumP2PRpc;
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@Getter
+public class PeerStat {
```

Comment:
Please add a `toString` for convenience.

Reply:
done